snapcraft/internal/common.py

```diff
@@ -183,7 +183,7 @@ def reset_env():
 
 
 def get_terminal_width(max_width=MAX_CHARACTERS_WRAP):
-    if os.isatty(sys.stdout.fileno()):
+    if os.isatty(1):
```

Comment:
I'm confused. Is this related to the `detach`? or something else I completely forgot?